### PR TITLE
Feature/add new marker

### PR DESCRIPTION
**Pull request by dzermas**
# Aruco marker support
## What?
Ability to add new aruco marker support when a new reflectance panel needs to be supported.
## Why?
We will be using different kinds/types of reflectance calibration panels. Each panel will have its own aruco marker id.
## PR Checklist
- [x] Merged latest master
- [x] Updated version number
- [x] Version numbers match between package ``__version__`` and *pyproject.toml*
- [x] All private git packages are at their newest version in both *pyproject.toml* and *environment.yml*
- [x] All git package version numbers match between *pyproject.toml* and *environment.yml*
- [x] dist/ImageryCorrector.exe has been rebuilt
## Breaking Changes

### CODE REVIEW

**Comment by dzermas** [comment]
Things to do:
* The 12.41 exiftool version download link is broken. Do we update to 12.50?
* Test has not happened yet.
* I do not know how to build an `exe`, @SamCWill will probably have to assist.

File: imgcorrect/zenith_co.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 """Define reflectance panel coefficients."""
 
-zenith_coefficients = [0.0] * 250
-zenith_coefficients.extend(
+sg3144_1_coefficients = [0.0] * 250
+sg3144_1_coefficients.extend(
```
Comment (zthorson) [inline imgcorrect/zenith_co.py:4]: I wonder if we should use the part numbers (either from the manufacturer, or from our system), so it's easier to correlate which coefficients are for which part.

**Review by zthorson — CHANGES_REQUESTED** [review]

File: imgcorrect/zenith_co.py (hunk 1 above)
Comment (dzermas) [reply imgcorrect/zenith_co.py:4]: @zthorson do you have a preference here? I can change the names to whatever.

Comment (zthorson) [reply imgcorrect/zenith_co.py:4]: I'm talking to Ryan to get a part number for it.  It sounds like the manufacturer part number is the same, but we are going to rev an internal part number.  So once I hear back from Ryan I'll let you know.

File: imgcorrect/corrections.py [hunk 2]
```diff
@@ -58,7 +58,11 @@ def _get_ils(row):
 def compute_reflectance_correction(image_df, calibration_df, ils_present):
     """Compute coefficient that will scale output values to known panel reflectance."""
 
-    def _get_band_coeff(row, coeffs):
+    def _get_band_coeff(row):
+        if row["aruco_id"] == 23:
+            coeffs = zenith_co.sg3144_1_coefficients
+        elif row["aruco_id"] == 63:
+            coeffs = zenith_co.sg3144_2_coefficients
```
[the remark this answers is not in the document]
Comment (zthorson) [reply imgcorrect/corrections.py:65]: Mostly display it to the log message to help if we have to support someone remotely that needs to run the script.

File: imgcorrect/zenith_co.py (hunk 1 above)
Comment (zthorson) [reply imgcorrect/zenith_co.py:4]: Ok, it sounds like we are just tracking this as batches internally in inventory.  So we'd refer to them as `sg3144_batch1` and `sg3144_batch2` which should match production.

Comment (dzermas) [reply imgcorrect/zenith_co.py]: No worries, it is straight forward to add as many as we need. It should be ready for delivery now. Need the executable. Documentation is finalized by @SamCWill.

**Review by zthorson — APPROVED** [review]
exe's are build and verified and should be ready for merging.

**Pull request merged**